core/src/test/java/com/radixdlt/api/core/TransactionStreamTest.java

```diff
@@ -331,15 +333,22 @@ public void test_previous_state_identifiers_and_proofs() throws Exception {
                       .fromStateVersion(1L));
       assertThat(firstPartResponse.getProofs()).isNull();
 
-      var firstPartResponseWithProofs =
-          getStreamApi()
-              .streamTransactionsPost(
-                  new StreamTransactionsRequest()
-                      .network(networkLogicalName)
-                      .includeProofs(true)
-                      .limit(100)
-                      .fromStateVersion(1L));
-      assertThat(firstPartResponseWithProofs.getProofs().size()).isEqualTo(13);
+      // An async "proofs pruning" is supposed to run over the genesis' epoch...
```

Comment:
How is this guaranteed? Is this because the test GC config runs quickly? This feels a little too implicit for my liking

Reply:
right :( there is awaitility (2x the GC period used in tests), but it still quietly assumes that GC completes quickly enough.

Together with the next comment to the same test, I now think it may be better to NOT "await until GC", just assert that `proofs.size() > 0`.